test/integration/helpers/project.js

```diff
@@ -0,0 +1,70 @@
+'use strict';
+
+var h = require('../helpers.js');
+
+// TODO: factor this out into a proper page object
+exports.visitCreatePage = function() {
+  h.goToPage('create-project');
+};
+
+exports.projectDetails = function() {
+  var timestamp = (new Date()).getTime();
+  var project = {
+    name:        'console-test-project-' + timestamp,
+    displayName: 'Console integration test Project ' + timestamp,
+    description: 'Created by integration tests'
+  };
+  return project;
+};
+
+exports.createProject = function(project, uri) {
+  for (var key in project) {
+    h.setInputValue(key, project[key]);
+  }
+  h.clickAndGo('Create', uri);
+};
+
+exports.deleteProject = function(project) {
+  h.goToPage('/');
+  var projectTile = element(by.cssContainingText(".project-info", project['name']));
+  projectTile.element(by.css('.fa-trash-o')).click();
+  h.setInputValue('confirmName', project.name);
+  var deleteButton = element(by.cssContainingText(".modal-dialog .btn", "Delete"));
+  browser.wait(protractor.ExpectedConditions.elementToBeClickable(deleteButton), 2000);
+  deleteButton.click();
+  h.waitForPresence(".alert-success", "marked for deletion");
+};
+
+exports.deleteAllProjects = function() {
```

Comment:
i think im just arguing that a test should know which projects it created and should be deleting those projects specifically, but if this is really the easiest way to do this im not totally opposed to it